trillian_log_api.proto

```diff
@@ -82,9 +82,9 @@ service TrillianLog {
   }
 
   // Returns the latest signed log root for a given tree. Corresponds to the
-  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  The server will
-  // return InvalidArgument if first_tree_size is greater than the
-  // LatestSignedLogRoot available to that server.
+  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  If
+  // first_tree_size > LatestSignedLogRoot, the server will return rpc code
```

Comment:
"first_tree_size > LatestSignedLogRoot" makes no sense -- one is an int, one is a structure.

Reply:
Revised to `first_tree_size > latest SignedLogRoot.tree_size`